x-pack/plugins/security_solution/public/resolver/store/middleware/current_related_event_fetcher.ts

```diff
@@ -35,10 +35,14 @@ export function CurrentRelatedEventFetcher(
     const indices = selectors.treeParameterIndices(state);
 
     const oldParams = last;
+    const newID = selectors.refreshCount(state);
     last = newParams;
 
     // If the panel view params have changed and the current panel view is the `eventDetail`, then fetch the event details for that eventID.
-    if (!isEqual(newParams, oldParams) && newParams.panelView === 'eventDetail') {
+    if (
+      (!isEqual(newParams, oldParams) && newParams.panelView === 'eventDetail') ||
```

Comment:
This can be done in a follow up PR, but you can either pull out the `newParams.panelView === 'eventDetail'` into a higher if, or cache it in a var